test/Ocelot.AcceptanceTests/Steps.cs

```diff
@@ -207,26 +221,26 @@ protected virtual void DeleteFiles()
             {
                 Console.WriteLine(e);
             }
-        }
-    }
-
-    protected virtual void DeleteFolders()
-    {
-        foreach (var folder in Folders)
-        {
-            try
-            {
-                var f = new DirectoryInfo(folder);
-                if (f.Exists && f.FullName != AppContext.BaseDirectory)
-                {
-                    f.Delete(true);
-                }
-            }
+        }
+    }
+
+    protected virtual void DeleteFolders()
+    {
+        foreach (var folder in Folders)
+        {
+            try
+            {
+                var f = new DirectoryInfo(folder);
+                if (f.Exists && f.FullName != AppContext.BaseDirectory)
+                {
+                    f.Delete(true);
+                }
+            }
```

Comment:
Ugly fake changes! 🙄

Reply:
I hope these changes are related to [this comment](https://github.com/ThreeMammals/Ocelot/pull/2156/files/bca0ba0edc6ca744c65bbebe8ae1a675a104dab4#r1792202279) because I don't remember doing that.

Reply:
The link is broken! It goes nowhere!